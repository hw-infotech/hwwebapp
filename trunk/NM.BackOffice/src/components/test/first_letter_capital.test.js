import {
  fireEvent,
  getAllByRole,
  getByTestId,
  getByText,
  render,
  screen,
  within,
} from "@testing-library/react";
import Enzyme from "enzyme";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import renderer from "react-test-renderer";
import  CapitalizeFirstLetter  from "../first_letter_capital";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });
describe("capitalletter", () => {
  //const captilize = jest.fn();
  //   test("First letter", () => {
  //     render();
  //     const myMock1 = jest.fn();
  //     const a = new myMock1();
  //   });
  test("Should render character name", () => {
    const capitalize=CapitalizeFirstLetter("this is the third react app of my life whatever ,i learned from this test is absolutly great")
    // let component = shallow(<CapitalizeFirstLetter />).getInstance();
    // //let tree = component.CapitalizeFirstLetter("the us");
    expect(capitalize).toBe("This is the third react app of my life whatever ,i learned from this test is absolutly great");
  });
});
