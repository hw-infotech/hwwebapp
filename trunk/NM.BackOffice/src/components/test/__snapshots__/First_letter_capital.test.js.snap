// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`capitalletter Should render character name 1`] = `undefined`;
